functions.yml

```diff
@@ -283,8 +295,9 @@ quality_control:
       args:
         - name: genotypes
           doc: The array of genotype structs
```

Comment:
An array of genotype structs with `depth` field

Reply:
Done